ansible/roles/common/tasks/debian.yml

```diff
@@ -4,6 +4,11 @@
     update_cache: True
     cache_valid_time: 3600
 
+- name: perform a dist-upgrade
+  apt:
+    upgrade: dist
+  when: common.upgrade_base is defined and common.upgrade_base|bool == True
```

Comment:
Is it typical Ansible to ensure something is defined before using it? Even if we set a default explicitly?

Reply:
I always add the `is defined` because it makes the code more resilient to user missteps. The way variables are namespaced right now makes it such that this could easily  become undefined.